util/log/log.go

```diff
@@ -17,7 +17,52 @@
 
 package log
 
-import "golang.org/x/net/context"
+import (
+	"strings"
+
+	"golang.org/x/net/context"
+)
+
+// Level Enum for log severity levels.
+type Level int
+
+// TODO(Bram): combine this with clog.severity, and rename severity.
```

Comment:
I'll have a new PR out to merge this with clog.severity.  I wrote most of this and didn't know it existed.